validation/src/test/scala/hmda/validation/rules/ts/validity/V155Spec.scala

```diff
@@ -12,15 +12,17 @@ class V155Spec extends PropSpec with PropertyChecks with MustMatchers with TsGen
   property("A valid email address for the institution must be provided") {
```

Comment:
This language is directly extracted from the edit checks document, it's the description of what this rule should do.